core/src/test/java/org/wso2/carbon/kernel/jmx/MBeanRegistratorTest.java

```diff
@@ -1,83 +0,0 @@
-/*
- * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.wso2.carbon.kernel.jmx;
-
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-import org.wso2.carbon.kernel.CarbonServerInfo;
-import org.wso2.carbon.kernel.Constants;
-import org.wso2.carbon.kernel.transports.TransportManager;
-
-import javax.management.InstanceNotFoundException;
-import javax.management.IntrospectionException;
-import javax.management.MBeanServer;
-import javax.management.MalformedObjectNameException;
-import javax.management.ObjectName;
-import javax.management.ReflectionException;
-
-/**
- * Unit test class for org.wso2.carbon.utils.MBeanRegistrator.
- *
- * @since 5.1.0
- */
-public class MBeanRegistratorTest {
```

Comment:
Check the possibility of using a separate MBean like runtime instead of transport manager and use that to test the MBean registration without removing the test itself.